src/sql/workbench/api/browser/mainThreadQueryEditor.ts

```diff
@@ -70,34 +72,25 @@ export class MainThreadQueryEditor extends Disposable implements MainThreadQuery
 		});
 	}
 
-	private static connectionProfileToIConnectionProfile(connection: azdata.connection.ConnectionProfile): IConnectionProfile {
```

Comment:
Ended up removing this completely - toIConnectionProfile already copies over all the values (including options) so we don't need this extra logic here.